stack_test.go

```diff
@@ -0,0 +1,22 @@
+package errors
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCaller(t *testing.T) {
+	s := NewStackTrace(0)
+	assert.NotEmpty(t, s.StackTrace)
+
+	assert.Equal(t, fmt.Sprintf("%s", s), "[stack_test.go testing.go asm_amd64.s]")
```

Comment:
Is portable meanwhile the stack is the same, based on:
https://github.com/pkg/errors/blob/master/example_test.go#L16-L36

Is being stable since a lot of time.